### PR TITLE
[ENO-1120] Add carthage and swiftlint

### PR DESCRIPTION
**Pull request by pavanbhagavathi**
CC: @rammareddy  @zac  @LaughlinElkind 

This PR is to install carthage and swiftlint on build machines.

### CODE REVIEW

File: setup/tasks/buildtools.yml [hunk 1]
```diff
@@ -19,6 +19,16 @@
...
 
+- name: Install Carthage
+  homebrew:
+    name: carthage
+    state: latest
+
+- name: Install SwiftLint
+  homebrew:
+    name: swiftlint
+    state: latest
```
Comment (rammareddy) [inline setup/tasks/buildtools.yml]: Looks good to me. Both Carthage and Swiftlint can be installed via Homebrew. And we are using the latest versions of both Carthage and Swiftlint in our project at this time. We might upgrade these tools to the latest versions from time to time. We would need to upgrade them on our new build machine as needed as well ...

Comment (pavanbhagavathi) [reply setup/tasks/buildtools.yml]: Yes, we can always have a specific version installed if we want with a PR.

**Comment by zac** [comment]
Sorry to request another tool, but would it be possible to add [bartycrouch](https://github.com/Flinesoft/BartyCrouch) in this PR?

It'd be something like this:

```YML
- name: Install Bartycrouch
  homebrew:
    name: bartycrouch
    state: latest
```

If we need a new ticket for this, let me know.

**Comment by pavanbhagavathi** [comment]
done @zac

**Pull request merged**